docs/sources/whatsnew/whats-new-next/README.md

```diff
@@ -0,0 +1,57 @@
+# Contribute to 'What's New in Grafana Cloud'
+
+To have a feature presented in [What's New in Grafana Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/), add an entry to the [`index.md`](./index.md) file in this directory.
+
+Use the following template, replace any `<VARIABLE>` with the appropriate text, explained later:
+
+```markdown
+## <FEATURE>
+
+<!-- <CONTRIBUTOR> -->
+<!-- <AVAILABILITY> -->
+
+<CLOUD AVAILABILITY> in Grafana <CLOUD OFFERING>
+
+<DESCRIPTION>
+```
+
+## _`CONTRIBUTOR`_
+
+The name of the contributor of the feature.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`AVAILABILITY`_
+
+One of:
+
+- OSS
```

Comment:
this is still a valuable field (so that we know which cloud features should also be added to the OSS release), but shouldn't be an absolute because a feature might only belong in cloud

Reply:
_then eve gets to the next paragraph :)_

Reply:
I think you are saying that it is also valid for their not to be an appropriate on-premise offering. I guess in such a case we should say to omit the comment? 

@imatwawana do you agree?